src/exchanges/adapters/ZeroExV2Adapter.sol

```diff
@@ -9,72 +9,85 @@ import "../libs/OrderTaker.sol";
 /// @author Melonport AG <team@melonport.com>
 /// @notice Adapter to 0xV2 Exchange Contract
 contract ZeroExV2Adapter is ExchangeAdapter, OrderTaker {
+    /// @notice Extract arguments for risk management validations of a takeOrder call
+    /// @param _encodedArgs Encoded parameters passed from client side
+    /// @return riskManagementAddresses needed addresses for risk management
+    /// - [0] Maker address
+    /// - [1] Taker address
+    /// - [2] Maker asset
+    /// - [3] Taker asset
+    /// - [4] Maker fee asset
+    /// - [5] Taker fee asset
+    /// @return riskManagementValues needed values for risk management
+    /// - [0] Maker asset amount
+    /// - [1] Taker asset amount
+    /// - [2] Fill amount
+    function extractTakeOrderRiskManagementArgs(
+        bytes memory _encodedArgs
+    )
+        public
+        view
+        override
+        returns (address[6] memory, uint256[3] memory)
+    {
+        address[6] memory riskManagementAddresses;
+        uint256[3] memory riskManagementValues;
+        (
+            address[4] memory orderAddresses,
+            uint256[7] memory orderValues,
+            bytes[2] memory orderData,
+        ) = __decodeTakeOrderArgs(_encodedArgs);
+
+        riskManagementAddresses = [
+            orderAddresses[0],
+            orderAddresses[1],
+            __getAssetAddress(orderData[0]),
+            __getAssetAddress(orderData[1]),
+            address(0),
+            address(0)
```

Comment:
Why aren't there maker/taker fee addresses for 0x v2? Shouldn't it ZRX, at least when there is a fee?